reference/5.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -45,12 +45,13 @@ This cmdlet was introduced in Windows PowerShell 3.0.
 Get the latest feeds from the PowerShell team's blog.
 
 ```PowerShell
-Invoke-RestMethod -Uri https://blogs.msdn.microsoft.com/powershell/feed/ |
+PS C:\>Invoke-RestMethod -Uri https://blogs.msdn.microsoft.com/powershell/feed/ |
```

Comment:
If you are adding `PS C:\>`, then 'PowerShell' language moniker should be removed, because it's not a valid powershell code. Also in this case, you can simple combine two codesnippets.

I remember discussing this with @juanpablojofre and one of the feedbacks he heard is that people want a copy-pasteable code-snippets.
Adding `PS C:\>` preamble makes it harder to use the help.